src/ms_filter.jl

```diff
@@ -0,0 +1,209 @@
+
+"""
+    struct ModifiedSincFilter{T<:RealQuantity} <: AbstractRadFIRFilter{}
+
+A [Modified-Sinc filter](https://doi.org/10.1021/acsmeasuresciau.3c00017)
```

Comment:
The links in the docstring direct to the Erratum, not the original article itself. Is that intended?